### DISCUSSION TITLE
Toggle marker - to automatically highlight text

### DISCUSSION THREAD

**Discussion by markky21** [omnivore-app/omnivore#2569] [Ideas]
Hi!

I have idea for new feature.
I'am using touch device with pen, but this could work with mouse. I would like to be able to toggle the button and when I select text (with pen or with mouse), I would like the text to be highlighted automatically (without having to select the highlight from the context menu).

What do you think about it?

**Comment by reagle** [comment]
Agreed, I can do this when using the website, but not the app.
